src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -790,7 +785,11 @@ CustomTextLayout::CustomTextLayout(IDWriteFactory1* const factory,
     {
         // Get the font fallback first
         ::Microsoft::WRL::ComPtr<IDWriteTextFormat1> format1;
-        RETURN_IF_FAILED(_format.As(&format1));
+        if (_format.As(&format1) != S_OK)
+        {
+            // If IDWriteTextFormat1 does not exist, return directly as this OS version doesn't have font fallback.
+            return S_OK;

```

Comment:
```suggestion
            return S_FALSE;
```

`S_FALSE` is a successful return code that is still indicative of nothing having been done (successfully)

Reply:
This will successfully pass `RETURN_IF_FAILED` :smile: